src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Swift/SwiftType.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.InteropServices.Swift
+{
+    /// <summary>
+    /// Indicates that the argument is the self context.
+    /// </summary>
+    public readonly struct SwiftSelf
```

Comment:
These should go through API review before this PR is merged.

Reply:
Thanks. I will open a new PR for the API changes and revert this.